src/components/calcite-combobox-item/calcite-combobox-item.scss

```diff
@@ -0,0 +1,66 @@
+:host,
+:host ul {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+// indentation on children
+:host .combobox-item-label.combobox-item-nested {
+  padding-left: $baseline * 3;
+}
+
+:host .combobox-item-label {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  min-width: 100%;
+  align-items: center;
+  color: var(--calcite-ui-text-3);
+  transition: $transition;
+  padding: var(--calcite-combobox-item-padding);
```

Comment:
this is not going to work in ie11 (using CSS variables set in the scope of the parent component)